Sources/ScryfallKit/Models/Card/Card.swift

```diff
@@ -156,7 +160,7 @@ public struct Card: Codable, Identifiable, Hashable, Sendable {
   /// A dictionary of links to other MTG resources
   public var relatedUris: [String: String]
   /// This card's release date
-  public var releasedAt: String
+  public var releasedAt: String?
```

Comment:
[Scryfall's API docs](https://scryfall.com/docs/api/cards#print-fields) don't list this as an optional, have you seen cards where this property is nil?